srcpkgs/pipewire/template

```diff
@@ -24,6 +24,11 @@ if [ "$XBPS_TARGET_NO_ATOMIC8" ]; then
 fi
 
 post_install() {
+	# alsa.conf scans /usr/etc/alsa/conf.d
+	mkdir -p ${DESTDIR}/usr/etc/alsa
+	mv ${DESTDIR}/usr/share/alsa/alsa.conf.d ${DESTDIR}/usr/etc/alsa/conf.d
+	rm -d ${DESTDIR}/usr/share/alsa
+
```

Comment:
I don't think it is appropriate to install these in `/usr/etc/alsa` by default. There is a legitimate interest in installing this package without forcing the default behavior or the specific device configuration that is included. Let people make symlinks in `/etc/alsa/conf.d` to the contents of `/usr/share/alsa/alsa.conf.d` if they wish to enable these defaults.

Reply:
Ok, I'll revert. But then we need to mention this in our docs I think, because upstream doesn't say anything about it (see https://gitlab.freedesktop.org/pipewire/pipewire/-/blob/master/README.md#running-alsa-applications). That's why I thought it should have worked ootb.

Reply:
nevermind, it is mentioned here: https://gitlab.freedesktop.org/pipewire/pipewire/-/blob/master/INSTALL.md#alsa-plugin

Reply:
but distros (e.g. fedora) seem to ship the symlinks

Reply:
Could INSTALLmd be `vdoc`'d?

@ahesford in what situation do you think one would one install `alsa-pipewire` and not want it to be on by default?

Reply:
it doesn't make sense to vdoc INSTALL.md imo, there's a lot of irrelevant info

Reply:
Because ALSA can be configured per-user and pipewire is designed to be run per-user, I can imagine situations where the package is installed to provide users with alsa emulation for pipewire *if they want it*, but not all users will want it.

This is probably more true on Void than other systems because launching the per-user daemon requires explicit construction of a per-user service management system. On systems like Fedora that use systemd, per-user services are possible by default.

If there is a consensus that the convenience of automatically enabling these options outweighs these concerns, then at least leave them in `/usr/share` and make symlinks in `/usr/etc`, so they can be disabled without destroying the contents.

Reply:
I don't want to block the update on this decision. We can think about adding the symlinks later.

Reply:
Fair enough.